root/place/edit_form.tt

```diff
@@ -46,15 +50,68 @@
   </div>
 
   <div class="documentation">
-    [%- area_bubble() -%]
+    <div class="bubble" id="name-bubble">
+      <p>
+        [% l('The {doc|name} is the place’s official name.',
+             { doc => { href => doc_link('Place#Name'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Place#Name'), target => '_blank' } }) %]
+      </p>
+    </div>
+
+    <div class="bubble" id="comment-bubble">
+      <p>
+        [% l('The {doc|disambiguation} field is used to help users distinguish between identically‐ or similarly‐named places.
+              It’s okay to leave it blank if it seems unlikely that there will be other places with similar names.',
```

Comment:
I'd expect disambiguations to mostly only be used for other places with similar names *in the same area*.

Reply:
Sure, makes sense. Maybe something like this?

> The disambiguation field is used to help users distinguish between places with similar names in the same area. Feel free to leave this blank if you aren't warned about possible duplicates in the same area.

Or would you prefer something stronger, like "Please leave this blank if ..."?